src/components/Dropdown/index.js

```diff
@@ -166,24 +123,41 @@ class Dropdown extends Component {
                   />
                 </div>
               </div>
-            </button>
+            </div>
+
+            <select
+              disabled={disabled}
+              className={selectClasses}
+              onChange={this.handleChange}
+              value={value}
+              autoFocus={autoFocus}
+              form={form}
+              name={name}
+              required={required}
+              style={{
+                // safari select height fix
+                webkitAppearance: 'menulist-button',
+              }}
+            >
+              {preventTruncate && (
+                <optgroup label={label || helpText || placeholder || ''}></optgroup>
```

Comment:
(To em dúvida no valor dessa label pegar o `label`. Mas não tenho sugestão melhor. Vamos ver no que vai dar)

Reply:
@klzns eu tava pegando do `placeholder` a princípio, mas achei que o `label` funciona melhor em mais casos.